ocw/lib/gke.py

```diff
@@ -0,0 +1,67 @@
+import os
+import kubernetes
+import googleapiclient.discovery
+from google.oauth2 import service_account
+from ocw.lib.gce import GCE
+from ocw.lib.k8s import clean_jobs
+
+
+class GKE(GCE):
+    __instances = {}
+
+    def __new__(cls, vault_namespace):
+        if vault_namespace not in GKE.__instances:
+            GKE.__instances[vault_namespace] = self = object.__new__(cls)
+            self.__gke_client = None
+            self.__kubectl_client = {}
+
+        return GKE.__instances[vault_namespace]
+
+    def gke_client(self):
+        if self.__gke_client is None:
+            self.private_key_data = self.get_data()
```

Comment:
you don't need this line . GKE is child of GCE which means that at this point `GCE.__init()` was already executed so `self.private_key_data` is already has data

Reply:
done